spark/src/test/scala/org/apache/comet/exec/CometAggregateSuite.scala

```diff
@@ -39,10 +42,7 @@ class CometAggregateSuite extends CometTestBase with AdaptiveSparkPlanHelper {
   import testImplicits._
 
   test("count with aggregation filter") {
-    withSQLConf(
-      CometConf.COMET_ENABLED.key -> "true",
-      CometConf.COMET_EXEC_SHUFFLE_ENABLED.key -> "true",
-      CometConf.COMET_SHUFFLE_MODE.key -> "jvm") {
```

Comment:
Instead of specifying explicit shuffle mode per test, we now run each test in multiple shuffle modes. See overloaded `test` method at the end of this file for details.